src/lib/data/creatables/gracefulOutfits.ts

```diff
@@ -1226,42 +1226,183 @@ export const gracefulOutfitCreatables: Createable[] = [
 	// Colossal Wyrm Agility (Varlamore graceful)
 	{
 		name: 'Varlamore graceful outfit',
-		inputItems: new Bank({
-			'Graceful crafting kit': 1,
-			'Graceful hood': 1,
-			'Graceful top': 1,
-			'Graceful legs': 1,
-			'Graceful gloves': 1,
-			'Graceful boots': 1,
-			'Graceful cape': 1
-		}),
-		outputItems: new Bank({
-			'Varlamore graceful hood': 1,
-			'Varlamore graceful top': 1,
-			'Varlamore graceful legs': 1,
-			'Varlamore graceful gloves': 1,
-			'Varlamore graceful boots': 1,
-			'Varlamore graceful cape': 1
-		})
+		inputItems: {
+			[itemID('Graceful crafting kit')]: 1,
+			[itemID('Graceful hood')]: 1,
+			[itemID('Graceful top')]: 1,
+			[itemID('Graceful legs')]: 1,
+			[itemID('Graceful gloves')]: 1,
+			[itemID('Graceful boots')]: 1,
+			[itemID('Graceful cape')]: 1
+		},
```

Comment:
Why change these?

Reply:
@nwjgit asked for these changes to make the code style consistent with other graceful commands.
 https://discord.com/channels/342983479501389826/648196527294251020/1296200249932779532

Reply:
This format of `{[itemID()]: 1}` is something we don't want to use anymore (although it still may exist in current code)

Reply:
All commands have been altered from `itemID` to `Bank`. Was easiest to just find and replace on the whole file. Also noted that dark graceful was missing, so added it.